app/helper/hedera/transaction-id.go

```diff
@@ -33,7 +33,7 @@ func ToMirrorNodeTransactionID(txId string) string {
 	return fmt.Sprintf(
 		"%s-%s-%s",
 		accId,
-		split[0],
+		fmt.Sprintf("%09s", split[0]),
```

Comment:
why is this change required

Reply:
Quite often there are timestamps with a lower-digit seconds, therefore an additional 0 appears in the representation, which causes differences which fail data retrieval.